frc_characterization/drive_characterization/data_analyzer.py

```diff
@@ -563,37 +580,44 @@ def setMeasurementDelay(delay):
                 STATE.controller_time_normalized.set(True),
                 STATE.controller_type.set("Talon"),
                 # https://phoenix-documentation.readthedocs.io/en/latest/ch14_MCSensor.html#changing-velocity-measurement-parameters
-                setMeasurementDelay(164),
+                # 100 ms sampling period + a moving average window size of 64 (i.e. a 64-tap FIR) = 100 ms + (64-1)/2 ms = 131.5 ms.
+                # See above for more info on moving average delays.
+                setMeasurementDelay(131.5),
             ),
             "Talon SRX (2020-)": lambda: (
                 STATE.max_controller_output.set(1),
                 STATE.period.set(0.001),
                 STATE.controller_time_normalized.set(True),
                 STATE.controller_type.set("Talon"),
+                # No known filtering/delay (other than CAN latency)
```

Comment:
The Talon SRX default latency is the same as the Spark MAX brushed default latency, iirc, so I don't think this is correct.

Reply:
You got me to reread the CTRE docs, and I now see that I misread them as only applying to the Falon FXes. However, [the docs](https://phoenix-documentation.readthedocs.io/en/latest/ch14_MCSensor.html#changing-velocity-measurement-parameters) seem to indicate that the Talon SRX delay is the same as the Talon FX delay (i.e. both are 131.5 ms.) I've pushed a change to fix this.